src/components/Hero/HeroSection.astro

```diff
@@ -0,0 +1,107 @@
+---
+import CountUp from "react-countup";
+import GridWrapper from "./GridWrapper";
```

Comment:
remove this. use tailwind grid classes. 
This is not required. same thing can be made using normal tailwind.

Reply:
done